cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-scaling.adoc

```diff
@@ -0,0 +1,259 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cloud-experts-deploying-application-scaling"]
+= Tutorial: Scaling an application
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-deploying-application-scaling
+:source-highlighter: pygments
+:pygments-style: emacs
+:icons: font
+
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 2024-04-10
+
+== Scaling
+
+You can manually scale your pods or you can automatically scale your pods by using the Horizontal Pod Autoscaler (HPA). You can also scale your cluster nodes.
+
+=== Manual pod scaling
+
+You can manually scale the pods and nodes of your applications:
+
+* by changing your replicaset and deployment definition 
```

Comment:
it would be more accurate to say "...replicaset _or_ deployment definition". As you don't have to modify both as youd generally be using one or the other